### ISSUE TITLE
EPEERINVALID error

### ISSUE THREAD

**Issue by Zacqary** [cyclejs/cycle-http-driver#12]
When using the latest @cycle/core, @cycle/dom, and @cycle/http together and running `npm install`, I get this error:
```
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "i"
npm ERR! node v0.10.35
npm ERR! npm  v2.13.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package @cycle/core@2.0.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer @cycle/dom@4.0.1 wants @cycle/core@2.x.x
npm ERR! peerinvalid Peer @cycle/http@1.0.3 wants @cycle/core@1.x.x
```
Manually changing this to 2.x.x solved the issue.

I don't want to pull request this change because I'm not sure if there's a specific reason why @cycle/http hasn't been updated yet. However, it seems to be working with @cycle/core 2.x.x for me, and it's otherwise breaking npm.

**Comment by staltz** [comment]
Fixing as we speak

**Comment by staltz** [comment]
Fixed with HTTP Driver v2.0
